modules/mod_ginger_rdf/controllers/controller_hydra.erl

```diff
@@ -17,7 +17,7 @@
 -include_lib("../include/rdf.hrl").
 
 -record(state, {
-    ontology :: atom(),
+    ontologies :: atom(),
```

Comment:
Should the type specifier be a list of atoms instead?

Reply:
Nice catch! Adding those types without actually having Dialyzer on CI is of course more for illustrative/documentary purposes, but still, it should be correct. 😄